src/cogent3/core/new_sequence.py

```diff
@@ -2596,42 +2636,48 @@ def copy(self, sliced: bool = False):
 
 
 @singledispatch
-def _coerce_to_seqview(data, seqid, alphabet) -> SeqViewABC:
+def _coerce_to_seqview(data, seqid, alphabet, offset) -> SeqViewABC:
```

Comment:
in the old sequence the new argument is called `annotation_offset`, otherwise the changes are the same

Reply:
Given the old module is destined for removal, I'm going to let that inconsistency fade gracefully into oblivion!